### ISSUE TITLE
Support for FL3 Plus?

### ISSUE THREAD

**Issue by blaqone** [j-kaltes/Juggluco#191]
Here FL3 Plus is available. Is there anything known if the sensor works with juggluco?

https://www.freestylelibre.de/produkte/freestyle-libre-3-plus-sensor.html

**Comment by j-kaltes** [comment]
Nothing is known.

**Comment by gui-dos** [comment]
The recent update of the the Libre 3 app for the Italian market to the version number 3.6.0 (i.e. https://www.apkmirror.com/apk/abbott-diabetes-care-inc/freestyle-libre-3-it/freestyle-libre-3-it-3-6-0-release/freestyle-libre-3-it-3-6-0-android-apk-download/) may be related since the packaging is trilingual.

**Comment by uschindler** [comment]
The only difference is that it works 15 days instead of 14 days. Some German Insurances already order them, mine (TKK) is still listing only Libre 3.

I think once somebody has the hardware available, we can test it with the app. Hopefully they only changed a few things, but in worst case it uses different encryption keys. BUT: There was no update of Libre 3 app until now, so it could be that the sensor protocol is unmodified except end date.

**Comment by gui-dos** [comment]
@uschindler:

Notice that the last week there was indeed a version bump to 3.6 for the Italian Libre 3 app both for iOS and Android, i.e. https://www.apkmirror.com/apk/abbott-diabetes-care-inc/ (the packaging is trilingual DE/FR/IT, in fact).

**Comment by blaqone** [comment]
I ordered one yesterday and will report.

**Comment by j-kaltes** [comment]
If what users wrote to me is right, Libre 2 PLUS was identical to Libre 2 with only the maximal wear duration extended to 15 days. In Great Britain they not even used the Gen 2 version of Libre 2 used in the United States, Canada, Australia, South Korea etc. 
If they made Libre 3 PLUS the same way, it would require minor modification.
A recent update of the Libre 3 app is totally irrelevant: They could have modified the app at the moment they started developing the new sensor, long before users would receive the sensor. This was the case with a modification of the Gen 2 Libre 2 sensor (US like Libre 2 sensor). They were modified without any notification to the user and the app was changed to make it possible a year before the users received the modified sensors.
Juggluco 8.1.6 is modified so that it uses a 15 days wear duration for Libre 3 PLUS.
https://www.juggluco.nl/Juggluco/download.html

**Comment by uschindler** [comment]
Should there anything change for Libre3 without plus, because diff of new version has some analysis of wear duration, so I expect it to have a slightly different guess on end date when I install it now.

The Libre3 I had always survived a few minutes longer than it told?

**Comment by uschindler** [comment]
Ok after scanning classical Libre3 it calculated same end time. So works fine. Let's see what reporter says.

**Comment by gui-dos** [comment]
@uschindler:

Yep, if you stop the "Trident" app some minutes before the 14-day formal expiration you can notice that the current Libre 3 keeps on advertising for another whole day, then its status switches from `05` (expired) to `06` (terminated) and the sensor shuts down by itself.

Otherwise, as soon as you run "Trident" on the 15th day, you can sniff that the app is sending a specific "shutdown" control command and the sensor is not detectable via BLE anymore.

For the Libre 2 I noticed that the custom command `1A 1D` also disable the sensor Bluetooth but it can be reactivated countless time with the successive NFC subcommand `A1 1E` ;-)

**Comment by blaqone** [comment]
With recent update FL3+ works perfectly fine.
